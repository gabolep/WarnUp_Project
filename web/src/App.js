import { Component } from 'react';

//components
import Navbar from './components/Navbar';
import Layout from './components/Layout';
import News from './components/News';
import Footer from './components/Footer';

class App extends Component {

  render(){
    return(
      <div>
        <Navbar/>
        <Layout>
          <News/>
        </Layout>
        <Footer/>
      </div>
    )
  }
  
}

export default App;
