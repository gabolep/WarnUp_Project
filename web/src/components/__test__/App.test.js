import { render, screen } from '@testing-library/react';
import ReactDOM from "react-dom";
import App from '../../App';
import renderer from "react-test-renderer";


test('renders without crashing', () => {
  const div = document.createElement("div");
  ReactDOM.render(<App />, div);

});
it("matches snapshot", () => {
  const tree = renderer.create(<App/>).toJSON();
  expect(tree).toMatchSnapshot();
})
