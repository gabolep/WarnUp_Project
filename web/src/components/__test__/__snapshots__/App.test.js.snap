// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <nav
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#303030",
        "boxShadow": "0 2px 3px rgb(0,0,0,0.1)",
        "color": "#fff",
        "display": "flex",
        "flexDirection": "row",
        "height": "220px",
        "justifyContent": "space-between",
        "padding": "0 50px",
        "position": "relative",
      }
    }
  >
    <div>
      <div
        style={
          Object {
            "fontSize": "4rem",
            "fontWeight": "630",
          }
        }
      >
        HN Feed
      </div>
      <div
        style={
          Object {
            "fontSize": "1.3rem",
            "fontWeight": "300",
            "paddingLeft": "5px",
          }
        }
      >
        We 
        &lt;3
         hacker news!
      </div>
    </div>
  </nav>
  <div
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#fff",
        "color": "#0A283E",
        "display": "flex",
        "flexDirection": "column",
      }
    }
  >
    <div
      style={
        Object {
          "width": "90%",
        }
      }
    >
      <div
        style={
          Object {
            "backgroundColor": "#fff",
            "color": "#000000",
            "cursor": "pointer",
            "marginTop": "40px",
            "position": "static",
          }
        }
      >
        <table
          style={
            Object {
              "margin": 0,
              "padding": 0,
              "width": "100%",
            }
          }
        >
          <tbody />
        </table>
      </div>
    </div>
  </div>
  <footer
    data-testid="footer"
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#303030",
        "bottom": "0",
        "color": "#fff",
        "flexDirection": "column",
        "height": "100px",
        "marginTop": "100px",
        "position": "absolute",
        "textAlign": "center",
        "width": "100%",
      }
    }
  >
    <h3
      data-testid="h3"
      style={
        Object {
          "fontSize": "1rem",
          "fontWeight": "630",
          "paddingTop": "20px",
          "textAlign": "center",
        }
      }
    >
      website created by Gabriel Escudero
    </h3>
  </footer>
</div>
`;
